ironfish-cli/src/commands/reset.ts

```diff
@@ -34,66 +32,49 @@ export default class Reset extends IronfishCommand {
 
   node: IronfishNode | null = null
 
-  peerNetwork: PeerNetwork | null = null
-
   async start(): Promise<void> {
     const { flags } = await this.parse(Reset)
 
-    let node = await this.sdk.node({ autoSeed: false })
-    await NodeUtils.waitForOpen(node, null, { upgrade: false, load: false })
+    let confirmed = flags.confirm
 
-    const backupPath = path.join(this.sdk.config.dataDir, 'accounts.backup.json')
+    const warningMessage =
+      `\n/!\\ WARNING: This will permanently delete your accounts. You can back them up by loading the previous version of ironfish and running ironfish export. /!\\\n` +
+      '\nHave you read the warning? (Y)es / (N)o'
 
-    if (fs.existsSync(backupPath)) {
-      this.log(`There is already an account backup at ${backupPath}`)
+    confirmed = flags.confirm || (await CliUx.ux.confirm(warningMessage))
 
-      const confirmed = await CliUx.ux.confirm(
-        `\nThis means this failed to run. Delete the accounts backup?\nAre you sure? (Y)es / (N)o`,
-      )
+    if (!confirmed) {
+      this.exit(1)
```

Comment:
If the user doesn't confirm a reset, I don't think the command necessarily failed. What do you think about logging some message and returning a 0?

Reply:
👍 Updated this